ui/tests/helpers/kv/kv-general-selectors.js

```diff
@@ -13,6 +13,12 @@ export const SELECTORS = {
   secretTab: (tab) => `[data-test-secrets-tab="${tab}"]`,
   emptyStateTitle: '[data-test-empty-state-title]',
   emptyStateMessage: '[data-test-empty-state-message]',
+  kvRow: '[data-test-kv-row]',
+  maxVersions: '[data-test-input="maxVersions"]',
```

Comment:
the data-test-input selectors could stay in this file if we generalize them, what are your thoughts on doing what I did above: 
```suggestion
  inputByAttr: (attr) => `[data-test-input="${attr}"]',
```

that way this test file doesn't get super bloated with each param having it's own selector